AGENTS.md

```
@@ -58,21 +66,129 @@ IT_DEPLOY_MODE=kustomize ginkgo run it
+
+# Build tasks
+uv run dev.py build
+```
+
+Python tooling uses uv for dependency management, ruff for linting. See `dev/README.md` for extending `dev.py`. Note: `buf` is not managed by `dev.py setup`; it's installed via `buf-action` in CI and must be installed separately for local development.
+
 ### Running Locally
 
-See [README.md](README.md) for instructions on running the service locally, including PostgreSQL setup and starting the gRPC server and REST gateway.
+Start PostgreSQL (containerized recommended):
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)